service/build.gradle

```diff
@@ -29,8 +29,12 @@ dependencies {
         implementation 'com.google.guava:guava:31.1-jre' // "-jre" for Java 8 or higher
     }
     implementation group: "com.google.guava", name:"guava"
+    implementation 'org.apache.taglibs:taglibs-standard-impl:1.2.5' // fixes taglibs vulnerability in javax.servlet:jstl:1.2
+    implementation 'org.json:json:20230227' // fixes json-java vulnerability
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.2' // fixes jackson-databind vulnerability from terra-common-lib:0.0.91-SNAPSHOT
+    implementation 'com.google.oauth-client:google-oauth-client:1.33.3' // fixes google oauth vuln from terra-common-lib:0.0.91-SNAPSHOT
```

Comment:
I'm normally a fan of comments, but these will kind of be outdated as soon as we merge.  I think these comments are helpful in the PR summary though, so that we can tie which library update fixed which specific vulnerability in _this_ PR.

Reply:
Okie dokie! I'll leave them in the PR description and remove them from the code